mobile/app/login.tsx

```diff
@@ -89,6 +90,8 @@ const LoginForm = ({
 
       <Typography>{t("paragraph")}</Typography>
 
+      {authError && <CredentialsError />}
```

Comment:
here we will use the error that we throw from the context instead of this

Reply:
done